ee/codegen/src/generators/nodes/inline-prompt-node.ts

```diff
@@ -135,7 +136,25 @@ export class InlinePromptNode extends BaseNode<
       );
     }
 
-    if (functionDefinitions.length > 0) {
+    const functionsAttribute = this.nodeData.attributes?.find(
+      (attr) => attr.name === "functions"
+    );
+
+    if (
+      functionsAttribute &&
+      this.shouldIncludeFunctionsAttribute(functionsAttribute)
```

Comment:
we don't need this helper, remove it. If tests fail bc we remove this check, let it fail and let me see